src/main/java/com/monumental/repositories/VerificationTokenRepository.java

```diff
@@ -10,6 +10,6 @@
 
     VerificationToken getByToken(String token);
 
-    List<VerificationToken> findAllByUser(User user);
+    List<VerificationToken> deleteAllByUserAndType(User user, VerificationToken.Type type);
```

Comment:
Idk if we need to have this return anything.

Reply:
Fixed